angular/core/services/thread_service.coffee

```diff
@@ -0,0 +1,20 @@
+angular.module('loomioApp').factory 'ThreadService', (Session, Records, ModalService, MuteExplanationModal, FlashService) ->
```

Comment:
I have a hunch this is actually 'DiscussionReaderService', but I'm not certain yet.